TurkGate/gateway.php

```diff
@@ -83,6 +86,11 @@
     }
     // Pass assignmentId to survey
     $surveyURL .= "assignmentID=$assignId";
+
+    // Pass along workerId if necessary
+    if ($_GET['passId'] == true) {

```

Comment:
`true` needs to have quotes around it. currently, this evaluates to true regardless of whether passId is true or false, meaning the workerid is passed even when instructed not to